charts/stable/seafile/values.yaml

```
@@ -32,15 +36,69 @@ workload:
             SEAFILE_ADMIN_EMAIL: me@example.com
             SEAFILE_ADMIN_PASSWORD: a_very_secret_password
             FORCE_HTTPS_IN_CONF: false
+            JWT_PRIVATE_KEY: a_very_very_very_very_secret_key
+        memcached:
+          enabled: true
+          imageSelector: memcached
+          stdin: true
+          tty: true
+          args:
+            - -u
+            - root
+          probes:
+            liveness:
+              port: 11211
+              type: tcp
+            readiness:
+              port: 11211
+              type: tcp
+            startup:
+              port: 11211
+              type: tcp
+      hostAliases:
+        - ip: "127.0.0.1"
+          hostnames:
+            - memcached
+configmap:
+  env:
+    enabled: true
+    data:
+      env: |
+        TIME_ZONE=UTC
```

```suggestion
        TIME_ZONE=UTC
```
This likely needs to be somehow hooked into tz setting though...